tests/queries.py

```diff
@@ -0,0 +1,52 @@
+import os
+from datetime import datetime
+
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Search, Q
+
+dt = datetime.now()
+todaydate = dt.strftime("%Y.%m.%d")
+current_index = 'kubernetes_cluster-'+todaydate
+
+def get_elastic_search_api():
```

Comment:
We have this function in test_bs.py
please add a #TODO to use the same function in test_bs.py and here